libcxx/test/std/utilities/memory/allocator.traits/allocator.traits.members/allocate_at_least.pass.cpp

```diff
@@ -45,16 +45,18 @@ struct has_allocate_at_least {
 constexpr bool test() {
   { // check that std::allocate_at_least forwards to allocator::allocate if no allocate_at_least exists
     no_allocate_at_least<int> alloc;
-    std::same_as<std::allocation_result<int*>> decltype(auto) ret =
-        std::allocator_traits<decltype(alloc)>::allocate_at_least(alloc, 1);
+    using AllocTraits = std::allocator_traits<decltype(alloc)>;
+    std::same_as<std::allocation_result<int*>, AllocTraits::size_type> decltype(auto) ret =
```

Comment:
The template argument seems misplaced.
```suggestion
    std::same_as<std::allocation_result<int*, AllocTraits::size_type>> decltype(auto) ret =
```

Reply:
Thanks!